backend/small_eod/tags/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from .models import Tag
+
+
+class TagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tag
+        exclude = ['id', ]
```

Comment:
Możesz to odwrócić i używać ```fields```? W utrzymaniu jest to przewidywalne.

Reply:
Zrobione